cirq-ft/cirq_ft/algos/arithmetic_gates_test.py

```diff
@@ -0,0 +1,317 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+
+import cirq
+import cirq_ft
+import numpy as np
+import pytest
+from cirq_ft.infra import bit_tools
+
+
+def identity_map(n: int):
+    return {i: i for i in range(2**n)}
+
+
+def test_less_than_gate():
+    qubits = cirq.LineQubit.range(4)
+    gate = cirq_ft.LessThanGate(3, 5)
+    op = gate.on(*qubits)
+    circuit = cirq.Circuit(op)
+    maps = {
```

Comment:
What is this map?  Maybe a more descriptive name or a line comment?

Reply:
Changed to `basis_map`. It corresponds to the `maps` parameter as expected by `cirq.testing.assert_equivalent_computational_basis_map` 

https://quantumai.google/reference/python/cirq/testing/assert_equivalent_computational_basis_map#args